lib/support/clone_vm.rb

```diff
@@ -428,12 +428,83 @@ def find_datacenter
       root_folder.childEntity.grep(RbVmomi::VIM::Datacenter).find { |x| x.name == datacenter }
     end
 
+    def customization_spec
```

Comment:
As far as I see it, the customization is only for Linux at the moment (Windows needs Sysprep/SysprepText calls). It could be a good idea to guard this with an `raise ... unless linux?` call here and point it out in Readme

Reply:
I like the functionality, thank you :) Just got those two remarks here.

I work in a similar area currently (using OVF environments) and might tweak some things later in your code when my current PR is done. This opens a lot of new opportunities :+1:

Reply:
@tecracer-theinen thank you! The only thing I am uncertain of is how to handle the platform detection. I have added a note to the README for now, which may suffice, but am open to additional thoughts or guidance on how that could be added.